internal/engine/upper.go

```diff
@@ -612,17 +617,17 @@ func eventContainsConfigFiles(manifest model.Manifest, e manifestFilesChangedAct
 	return false
 }
 
-func getNewManifestFromTiltfile(ctx context.Context, name model.ManifestName) (model.Manifest, error) {
+func getNewManifestFromTiltfile(name model.ManifestName) (model.Manifest, *newManifestErr) {
 	t, err := tiltfile.Load(tiltfile.FileName, os.Stdout)
 	if err != nil {
-		return model.Manifest{}, err
+		return model.Manifest{}, newManifestErrf(err.Error())
```

Comment:
teeeechnically we're losing some error history here by not wrapping this... do we care?